api/src/main/java/org/openmrs/module/aijar/dataintegrity/BasePatientRuleDefinition.java

```diff
@@ -41,6 +49,39 @@ public String getExposedInfantNumber(Patient patient) {
 		PatientIdentifierType pit = patientService.getPatientIdentifierTypeByUuid(PatientIdentifierTypes.EXPOSED_INFANT_NUMBER.uuid());
 		return patient.getPatientIdentifier(pit) == null ? "" : patient.getPatientIdentifier(pit).toString();
 	}
+
+	/**
+	 * TB number for the patient
+	 * @param patient
+	 * @return
+	 */
+	public String getTbNumber(Patient patient) { //To be done
```

Comment:
A patient can have multiple TB numbers so you need to pass some more information here to get the TB number - maybe a date or the TB Summary page encounter not go through Programs